help/help.md

```diff
@@ -0,0 +1,130 @@
+% POSTFIX(1) Container Image Pages
+% Petr Hracek
+% September 19, 2017
+
+# NAME
+{{ spec.envvars.name }} - {{ spec.short_description }}
+
+# DESCRIPTION
+{{ spec.description }}
+
+# USAGE
+To get the {{ spec.envvars.name }} container image on your local system, run the following:
+
+    docker pull hub.docker.io/modularitycontainers/{{ spec.envvars.name }}
+
+Commands for testing {{ spec.envvars.name }} docker container:
+
+```$ telnet localhost 25```
+
+```HELO test.localhost```
+
+```MAIL FROM:<yourname>@<domain.com>```
+
+```RCPT TO:<demo@localhost>```
+
+```DATA``` and type whatever you want
+```
+Subject: My testing docker container image
+
+Hi, Testing message
+regards
+Docker
+.
+```
+
+## How to test the {{ spec.envvars.name }} mail server with TLS
+
+Command for testing {{ spec.envvars.name }} docker container with
+enabled TLS is ```openssl```.
+
+Telnet can not be used because of communication is encrypted.
+
+```
+$ openssl s_client -starttls smtp -crlf -connect localhost:25
+```
+
+## How to test the {{ spec.envvars.name }} with IMAP enabled
+
+Command for testing {{ spec.envvars.name }} docker container
+dovecot is ```openssl```.
+
+Telnet can not be used because of communication is encrypted.
+
+Testing {{ spec.envvars.name }} service with ```openssl```
+
+```
+$ openssl s_client -debug -starttls smtp -crlf -connect localhost:587
```

Comment:
Since you wrote it so nicely here, can we have one test for TLS connection?

Reply:
https://github.com/container-images/postfix/issues/13